api/routes/storeRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+
+const { storeController } = require('../controllers')
+
+const router = express.Router()
+
+router.get('', storeController.getAllStores)
+router.get('/storeType/:storeType', storeController.getStoreByType)
```

Comment:
```suggestion
router.get('/storeType/:storeType', storeController.getStoreByType)
```
storeType인가요 storeTypeId인가요??

Reply:
storeType  입니다 ..!

Reply:
다시 반영 필요